### PR TITLE
CLOUD-861 prune cloudbreak events/notifications

### PR DESCRIPTION
**Pull request by biharitomi**
@keyki or @schfeca75 please review

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/core/flow/service/AmbariClusterFacade.java [hunk 1]
```diff
@@ -291,24 +243,15 @@ public FlowContext downscaleCluster(FlowContext context) throws CloudbreakExcept
         Stack stack = stackService.getById(actualContext.getStackId());
         Cluster cluster = clusterService.retrieveClusterByStackId(stack.getId());
         MDCBuilder.buildMdcContext(cluster);
-
-        stackUpdater.updateStackStatus(stack.getId(), UPDATE_IN_PROGRESS);
+        String statusReason = String.format("Scaling down the Ambari cluster.", actualContext.getHostGroupAdjustment().getScalingAdjustment());
```
Comment (keyki) [inline core/src/main/java/com/sequenceiq/cloudbreak/core/flow/service/AmbariClusterFacade.java]: missing %s or something to show the adjustment

**Comment by doktoric** [comment]
do not merge pls

**Comment by biharitomi** [comment]
why?

**Comment by doktoric** [comment]
we will discuss it on monday

**Pull request merged**